packages/zowe-explorer/package.json

```diff
@@ -1794,7 +1794,10 @@
     "madge": "madge -c --no-color --no-spinner --exclude __mocks__ --extensions js,ts src/",
     "pretty": "prettier --write .",
     "generateLocalization": "pnpm dlx @vscode/l10n-dev export --o ./l10n ./src && node ./scripts/generatePoeditorJson.js",
-    "copy-secrets": "node ./scripts/getSecretsPrebuilds.js"
+    "copy-secrets": "node ./scripts/getSecretsPrebuilds.js",
+    "build:nlsExtract": "pnpm dlx @vscode/l10n-dev export --outDir ./l10n ./src",
```

Comment:
There may be some duplication in npm scripts here 😋 Since we already run `generateLocalization` script which exports l10n during prepublish.